### ISSUE TITLE
Cant set JobId for recognize from scheduler

### ISSUE THREAD

**Issue by lezag** [naptha/tesseract.js#399]
Is there any possibility for set an ID for worker job from by using addJob()?

**Comment by miguelm3** [comment]
I ran on this problem recently. Actually now I was able to set a name on the scheduler:
```
scheduler.addJob('recognize', dataFile, '/*I don´t know what goes here */', 'JobNameMiguel')
```
But you are only able to see if you activate the debug... (screenshot added) because the logger that you register doesn't access that info, only to the internal JobId.
``` 
const scheduler = tjs.createScheduler();
tjs.setLogging(true);
``` 
![Logger-vs-Internal](https://user-images.githubusercontent.com/10519208/99718652-f5f08980-2a89-11eb-8126-c5852bb723d4.png)

I plan to fork the repo for personal use and make a pull request to have access to this with something like userJobId.

**Comment by Balearica** [comment]
Closing issue as this appears to have been resolved by #496.
